mautic/mautic
# [UI] Fix filters on contact details weren't usable on mobile

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



| Before                                      | After
| ------------------- | ----------------------
| ![image](https://github.com/mautic/mautic/assets/116097999/20b1e0d9-2a37-48cc-bdc2-469b9af97d65)  | ![image](https://github.com/mautic/mautic/assets/116097999/891eb470-dcf8-4620-9e70-1ee01e529801)


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Contacts > Any contact > See History and Audit log
